modules/javafx.graphics/src/main/java/javafx/animation/Interpolator.java

```diff
@@ -95,7 +95,7 @@ public String toString() {
     };
 
     /**
-     * Creates a piecewise-linear interpolator with the specified control points.
+     * Returns a piecewise-linear interpolator with the specified control points.
```

Comment:
Question: do we need to specify that a new instance is returned each time (as opposed to some sort of caching as with `Integer.valueOf`)?

Reply:
I think usually only the stronger guarantee is mentioned explicitly (that it is a _new_ instance).